README.md

```diff
@@ -30,6 +30,13 @@ npm i && npm start
 The above will eventually output a ZIM file to ```dist/```
 
 ## Command line arguments
+`--withoutLanguageVariants` uses to exclude languages with County variant. For example `en_ca` will not be present in zim with this argument.
```

Comment:
is that not "en-CA"?

Reply:
From the phet site, we are getting "en_CA"